packages/core/src/Chip/ChipContentWrapper.tsx

```diff
@@ -78,6 +98,8 @@ const ChipContentWrapper: React.FC<IChipContentWrapper> = styled(Box)`
   position: relative;
   border-radius: 2px;
   white-space: nowrap;
+  ${({ theme, hasChildren }) => applySizes(getSizes({ hasChildren }), undefined, theme.mediaQueries)};
+  ${applyVariations('ChipContentWrapper', variations)};
```

Comment:
Line 102 is my change, and 101 is the result of reformatting after merging with main, and not part of my changes.